packages/components/molecules/f-promotions-showcase/src/components/PromotionsShowcase.vue

```diff
@@ -1,32 +1,224 @@
+                        {{ line.text }}
+                    </p>
+                </div>
+            </component>
+        </div>
     </div>
 </template>
 
 <script>
+import { STYLE_TEXT } from '../constants';
+
+function isString (item) {
+    return typeof (item) === 'string'
+        || item instanceof String;
+}
+
+function isFunction (item) {
+    return typeof item === 'function';
+}
 
 export default {
     name: 'PromotionsShowcase',
+
     components: {},
+
     props: {
+        items: {
+            type: Array,
+            default: () => []
+        }
+    },
+
+    methods: {
+        dispatchClick (item, event) {
+            return isFunction(item.link) && item.link(event);
+        },
+
+        getLines (lines) {
+            return lines.map(line => (isString(line)
+                ? {
+                    text: line,
+                    style: STYLE_TEXT
+                }
+                : line));
+        },
+
+        isFunction,
+
+        isVueComponent (component) {
+            return component?.render instanceof Function
+                || isString(component?.template)
+                || (component instanceof Function && component.name === 'VueComponent');
+        },
+
+        itemTag (item) {
+            if (isString(item.link) || isString(item.link?.href)) {
+                return 'a';
+            }
+
+            return isFunction(item.link)
+                ? 'button'
+                : 'div';
+        },
+
+        linkAttrs (item) {
+            if (isFunction(item?.link)) {
+                return false;
+            }
+
+            return isString(item?.link)
+                ? { href: item?.link }
+                : { ...item?.link };
+        }
     }
 };
 </script>
 
 <style lang="scss" module>
 
+$promotionsItems-borderRadius                        : $radius-rounded-c;
+
 .c-promotionsShowcase {
+    background-color: $color-support-brand-02;
+    padding: spacing(x2);
+}
+
+.c-promotionsShowcase--inner {
     display: flex;
+    flex-direction: column;
     justify-content: center;
-    min-height: 80vh;
-    width: 80vw;
-    margin: auto;
-    border: 1px solid $color-red;
-    font-family: $font-family-base;
-    @include font-size(heading-m);
+    background-color: $color-container-default;
+
+    box-shadow: 0 4px 6px       rgba(27, 35, 36, 0.02),
+                0 2px 12px -2px rgba(27, 35, 36, 0.08),
+                0 3px 6px       rgba(27, 35, 36, 0.06);
+
+    border-radius: $promotionsItems-borderRadius;
+
+    @include media ('>=wide') {
+        flex-direction: row;
+    }
+}
+
+.c-promotionsShowcase--item {
+    display: flex;
+    flex-direction: row;
+    padding: spacing(x2);
+    text-decoration: none;
+
+    // Override button styles
+    border: none;
+    background: transparent;
+    font-family: inherit;
+    text-align: inherit;
+
+    @include media ('>=wide') {
+        flex-grow: 1;
+    }
+
+    &:not(*:first-child) {
```

Comment:
If I understand what you're doing here correctly, you could use a sibling selector here as they are a little easier to understand?

```suggestion
    & + .c-promotionsShowcase--item {
```

Reply:
Yep - my intention was to select all but the first within a container. That makes sense, will action.